boletin-de-la-sociedad-geologica-mexicana.csl

```diff
@@ -0,0 +1,312 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" and="text" delimiter-precedes-et-al="always" delimiter-precedes-last="always" name-as-sort-order="all" demote-non-dropping-particle="never" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Boletín de la Sociedad Geológica Mexicana</title>
+    <id>http://www.zotero.org/styles/boletin-de-la-sociedad-geologica-mexicana</id>
+    <link href="http://www.zotero.org/styles/boletin-de-la-sociedad-geologica-mexicana" rel="self"/>
+    <link href="http://www.zotero.org/styles/ecology-letters" rel="template"/>
```

Comment:
I guess you used a different style as a base style based on the documentation link. Please change that.
Equally, provide a link to the reference guidelines in line 8.